docs/cep/query-guide/functions/unitconversion/kgToStones.md

```diff
@@ -28,7 +28,7 @@ In this example, the `unitconversion:kgToStones()` function converts a value of
 
 ```sql
 CREATE STREAM InputStream (mass_kg double);
-CREATE STREAM OutputStream (mass_kg double, mass_stones double);
+CREATE SINK STREAM OutputStream (mass_kg double, mass_stones double);
```

Comment:
There is nothing called "unitconversion:kgToStones" we have instead "unitconversion:kgTost" to convert into imperial stones.

Reply:
kgToStones is right here: https://www.macrometa.com/docs/cep/query-guide/functions/unitconversion/kgToStones

Reply:
@diana-macrometa that is incorrect, it is actually kgTost.. you can run and check

Reply:
@StoyanTc currently, we don't have any extension for kg to imperial stones..
The extension here uses "kgTost". but the case insensitiveness will reach out to kgToST (Kg to US tons)
https://github.com/Macrometacorp/c8cep/blob/921969e22f988ee3318850dcea0d7c4c60da17ba/engine/src/main/java/co/macrometa/c8cep/extension/execution/unitconversion/mass/KilogramToStone.java
https://github.com/Macrometacorp/c8cep/blob/921969e22f988ee3318850dcea0d7c4c60da17ba/engine/src/main/java/co/macrometa/c8cep/extension/execution/unitconversion/mass/KilogramToUSTon.java

So it seems we need to make code changes here, changing the extension name for kg to imperial stones to actually "kgToStones" instead of "kgTost" which conflicts with "kgToST" (US tons) .. and probably take the page down an d bring it back later once code changes are done?
What do you think?

Reply:
@koshy-thomas - Agreed. Please file a ticket. We'll fix this into the code.

Reply:
@StoyanTc @diana-macrometa Ticket filed for fixing this:
https://macrometa.atlassian.net/browse/CEP-559

Reply:
So...which page did you want me to take down? kgToStones?

Reply:
Yes kgToStones and charFrequency but then we also need to add them back when they are fixed in code

Reply:
Not a problem. Fun fact: Any file with an underscore in front of it is not included in the documentatation build. So I can temporarily hide things quite easily.